frontend/src/components/Interacted.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+const Interacted = ({ user, viewInteracted }) => {
```

Comment:
Could you change the name to make it more clearer? Currently 'Interacted' isn't clear enough to understand what the interatcion is for.